### PR TITLE
bump knative and hakn to 1.8.0

### PR DESCRIPTION
**Pull request by cpanato**
- bump knative and hakn to 1.8.0

### CODE REVIEW

File: setup-knative-eventing/action.yaml [hunk 1]
```diff
@@ -9,9 +9,9 @@ description: |
 inputs:
   version:
     description: |
-      The version of Knative to install, e.g. 1.5.0
+      The version of Knative to install, e.g. 1.8.0
     required: false
-    default: 1.5.0
+    default: 1.8.0
```
Comment (mattmoor) [inline setup-knative-eventing/action.yaml:14]: I think this broke @vaikas in scaffolding?

Comment (k4leung4) [reply setup-knative-eventing/action.yaml:14]: once https://github.com/sigstore/scaffolding/pull/431 lands, we can bump the version without concern of breaking scaffold

**Comment by k4leung4** [comment]
we should drop testing on 1.22.x, as knative 1.8 uses the k8s 0.25.x client lib, meaning it no longer supports 1.22.x

File: setup-knative-eventing/action.yaml (hunk 1 above)
Comment (k4leung4) [reply setup-knative-eventing/action.yaml:14]: once https://github.com/sigstore/scaffolding/pull/431 lands, scaffold will be pinned to 1.7.x, and using latest.

**Review by k4leung4 — APPROVED** [review]

**Pull request merged**